src/pybind/mgr/rbd_support/mirror_snapshot_schedule.py

```diff
@@ -133,6 +259,22 @@ def load_pool_images(self, ioctx, images):
             for namespace in namespaces:
                 images[pool_id][namespace] = {}
                 ioctx.set_namespace(namespace)
+                found, error, updated = self.watchers.status(pool_id, namespace)
```

Comment:
Nit: perhaps this can be cleaned up to a method that just returns `updated`. If the watcher does not exist, it registers it and returns `true`. If the watcher had an error, it unregisters the old one, registers a new one, and returns `true`. Else, it updates the epoch time and returns the current `updated` status.